### DISCUSSION TITLE
配置环境后在主线程无效

### DISCUSSION THREAD

**Discussion by Pangchengqiu12** [alex8088/electron-vite#487] [Q&A]
![Snipaste_2024-04-25_15-58-17](https://github.com/alex8088/electron-vite/assets/85088770/803a0fb0-e9e3-4c9c-9c44-be7eee0adc0c)
配置了环境变量在主线程中答应出来的是undefined

**Comment by alex8088** [comment]
请查阅文档，环境变量章节，并升级electron-vite版本到2.0及以上

**Reply by Pangchengqiu12** [reply]
是最新版了  不知道是否提供个demo
